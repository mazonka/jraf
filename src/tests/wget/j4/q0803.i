command=jw md 0 /.jraf.sys/users : login admim@server.com * : put f82d3bb17612458c /x 0 1 Kg==