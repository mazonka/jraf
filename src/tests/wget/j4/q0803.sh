#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
echo ""
echo "=="
pushd $wr >/dev/null
ls -1
popd >/dev/null
rm -rf $wr



