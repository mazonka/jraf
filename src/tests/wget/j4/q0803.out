OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf? : JRAF_FAIL denied
==
home
